ui/DesignSystem/Component/NotificationFaucet.svelte

```diff
@@ -19,7 +19,7 @@
     const id = Math.random();
     messages = [
       ...messages,
-      { id: id, message: message.text, level: message.level }
+      { id: id, message: message.text, variant: message.variant }
```

Comment:
@juliendonck I would keep it as `level` here and only change it in the prop for `Notification`, otherwise you will have to change it where ever a notification is created. I also think `level` in this part of the code makes more sense